src/Bridge/Symfony/Routing/ApiLoader.php

```diff
@@ -99,16 +99,19 @@ private function addRoute(RouteCollection $routeCollection, string $resourceClas
             throw new RuntimeException('Either a "route_name" or a "method" operation attribute must exist.');
         }
 
-        if (isset($operation['controller'])) {
-            $controller = $operation['controller'];
-        } else {
-            $actionName = sprintf('%s_%s', strtolower($operation['method']), $collection ? 'collection' : 'item');
-            $controller = self::DEFAULT_ACTION_PATTERN.$actionName;
-        }
+        $collectionString = $collection ? 'collection' : 'item';
+
+        $actionName = sprintf('%s_%s', $operationName, $collectionString);
+        $controller = $operation['controller'] ?? self::DEFAULT_ACTION_PATTERN.sprintf('%s_%s', strtolower($operation['method']), $collectionString);
```

Comment:
With my code:

```php
$actionName = sprintf('%s_%s', strtolower($operationName), $collection ? 'collection' : 'item');
$controller = $operation['controller'] ?? self::DEFAULT_ACTION_PATTERN.$actionName;
```

This is supported:

```php
/**
 * @Resource(itemOperations={
 *     "get"={"method"="GET"},
 * })
 */
```

while this is not:

```php
/**
 * @Resource(itemOperations={
 *     "my_get_op"={"method"="GET"},
 * })
 */
```

but you could specify it explicitly:

```php
/**
 * @Resource(itemOperations={
 *     "my_get_op"={"method"="GET", "controller"="api_platform.action.get_item"},
 * })
 */
```